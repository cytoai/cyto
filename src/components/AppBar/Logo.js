import React from 'react';
import styles from './PrimaryAppBar.css';
import { withStyles } from '@material-ui/core/styles';
import { Typography } from '@material-ui/core';

export default function Logo() {
  return (
    <Typography variant="h6" color="inherit">
      Logo
    </Typography>
  );
}

withStyles(styles, { withTheme: true })(Logo);
