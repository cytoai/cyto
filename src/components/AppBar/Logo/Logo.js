import React from 'react';
import { Typography } from '@material-ui/core';

export default function Logo() {
  return (
    <Typography variant="h6" color="inherit">
      Logo
    </Typography>
  );
}
