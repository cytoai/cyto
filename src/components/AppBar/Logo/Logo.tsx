import * as React from 'react';
import { Typography } from '@material-ui/core';
import { useTranslation } from 'react-i18next';

const Logo = () => {
  const { t, i18n } = useTranslation();

  return (
    <Typography variant="h6" color="inherit">
      {t('Logo')}
    </Typography>
  );
};

export default Logo;
