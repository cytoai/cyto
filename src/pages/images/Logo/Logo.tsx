import * as React from 'react';
import { Typography } from '@material-ui/core';
import { useTranslation } from 'react-i18next';

export const Logo = () => {
  const { t: translation } = useTranslation();

  return (
    <Typography variant="h6" color="inherit">
      {translation('Piximi')}
    </Typography>
  );
};
