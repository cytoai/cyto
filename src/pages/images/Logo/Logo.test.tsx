import * as React from 'react';
import { shallow } from 'enzyme';
import Logo from './Logo';

describe('Logo', () => {
  it('should render', () => {
    const component = shallow(<Logo />);

    expect(component).toMatchSnapshot();
  });
});
