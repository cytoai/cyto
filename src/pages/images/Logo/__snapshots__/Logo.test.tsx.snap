// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render 1`] = `
<WithStyles(Typography)
  color="inherit"
  variant="h6"
>
  Logo
</WithStyles(Typography)>
`;
